lib/cisco_node_utils/cmd_ref/vpc.yaml

```diff
@@ -194,16 +194,16 @@ system_mac:
   set_value: "<state> system-mac <mac_addr>"
   default_value: ''
 
-system_priority: 
+system_priority:
```

Comment:
We should probably add a step to Travis-CI to check for trailing whitespace in .yaml files (only; rubocop covers whitespace in .rb files), as these keep creeping back in.